File: Atividade Prática 1/main/Professores.java
public class Professor {

    private String nome;
    private String horario;
    private String periodo;
    private double sala;
    private double predio;

    public Inimigo (String nome, String horario, String periodo, double sala, double predio){
        this.nome = nome;
        this.horario = horario;
        this.periodo = periodo;
        this. sala = sala;
        this.predio = predio;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public double getHorario() {
        return horario;
    }

    public void setHorario(String horario) {
        this.horario = horario;
    }

    public String getPeriodo() {
        return periodo;
    }

    public void setPeriodo(String periodo) {
        this.periodo = periodo;
    }

    public String getSala() {
        return sala;
    }

    public void setSala(String sala) {
        this.sala = sala;
    }

    public String getPredio() {
        return periodo;
    }

    public void setPredio(String predio) {
        this.predio = predio;
    }
}

File: Lab/Tarefa2/main/Professores.ts
class Professor {
    private nome: string;
    private horario: string;
    private periodo: string;
    private sala: number;
    private predio: number;

    constructor(nome: string, horario: string, periodo: string, sala: number, predio: number) {
        this.nome = nome;
        this.horario = horario;
        this.periodo = periodo;
        this.sala = sala;
        this.predio = predio;
    }

    public getNome(): string {
        return this.nome;
    }

    public setNome(nome: string): void {
        this.nome = nome;
    }

    public getHorario(): string {
        return this.horario;
    }

    public setHorario(horario: string): void {
        this.horario = horario;
    }

    public getPeriodo(): string {
        return this.periodo;
    }

    public setPeriodo(periodo: string): void {
        this.periodo = periodo;
    }

    public getSala(): number {
        return this.sala;
    }

    public setSala(sala: number): void {
        this.sala = sala;
    }

    public getPredio(): number {
        return this.predio;
    }

    public setPredio(predio: number): void {
        this.predio = predio;
    }
}
